docs/_index.md

```diff
@@ -403,12 +403,13 @@ plugins(
 	)
 ```
 
-#### prezto
+#### Prezto
 
 ```zsh
 poetry completions zsh > ~/.zprezto/modules/completion/external/src/_poetry
 ```
 
 {{% note %}}
 You may need to restart your shell in order for these changes to take effect.
```

Comment:
I assume this note was meant for all types of shells. Maybe, we should put it at the top (before the sections for indiviual shells).